docs/ycsb.md

```diff
@@ -30,6 +30,8 @@ spec:
         - workloadb
       options_load: '-p mongodb.url="mongodb://mongo/ycsb?"' #passed as is to ycsb when loading database
       options_run: '-p mongodb.url="mongodb://mongo/ycsb?" -threads 10 -target 100'
```

Comment:
@aakarshg  Swapping single quotes by double quotes like here, at tests/test_crs/valid_ycsb-mongo.yaml should work

```suggestion
      options_load: "-p mongodb.url='mongodb://mongo/ycsb?'"
      options_run: "-p mongodb.url='mongodb://mongo/ycsb?' -threads 10 -target 100"
```